src/extension.ts

```diff
@@ -60,6 +60,9 @@ class RunOnSaveExtension {
 			var child = exec(cfg.cmd, this._execOption);
 			child.stdout.on('data', data => this._outputChannel.append(data));
 			child.stderr.on('data', data => this._outputChannel.append(data));
+			child.on('error', (e) => {
+				this.showOutputMessage(e);
```

Comment:
This shows a TypeScript compiler error. You'll probably want to wrap this in `String()`

Reply:
Fixed